content/webapp/pages/stories.tsx

```diff
@@ -99,12 +101,18 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     // would it be faster to skip all the fetchLinks?  Is that possible?
     const storiesPagePromise = fetchPage(client, prismicPageIds.stories);
 
+    const featuredSerial =
+      await client.client.getSingle<FeaturedSerialPrismicDocument>(
+        'featured-serial'
+      );
```

Comment:
Do we have to `await` here, or can we do something with chaining?

```
const featuredSerialPromise = fetchSingle(…)
const featuredSeriesArticlesQueryPromise = featuredSerialPromise.then(…)
```

rather than having a blocking `await`, which is quite ugly and potentially slow.

Also, can you replace this use of the naked client with a `fetchSingle()` method or similar, to follow the pattern we've used elsewhere? It enforces good behaviour in the client.

Reply:
I don't agree with the first point here fwiw - if we're talking about ugliness, I think the `then` is worse, and the second fetch is dependent on the first: it's annoying that we can't do it in parallel, but it is the case and we should be using the `await` for that (there is no performance difference, and it makes the code much more legible).